core/src/test/scala/integration/kafka/api/SaslPlainSslEndToEndAuthorizationTest.scala

```diff
@@ -1,169 +0,0 @@
-/**
```

Comment:
Why are we deleting this test? Is this covered by another test class?

Reply:
The only test case `testAcls` uses `zkClient` directly. Do we need to update to kraft? Thanks.